sources/licenses/Licenses.yml

```
@@ -45,12 +48,36 @@ swap:
   CC[ -]?BY[ -]?SA[ -]?4(?:\.0)?: CC BY-SA 4.0
   CC[ -]?[O0]: CC0
   CC[ -]?0[ -]?1(?:\.0)?: CC0 1.0
-  # GNU Licences
+  # GNU licences
   AGPL: AGPL
   AGPL[ -]?v?3(?:\.0)?: AGPLv3
   GPL(?!\.org): GPL
   GPL[ -]?v?2(?:\.0)?: GPLv2
-  GPL[ -]?v?3(?!\+): GPLv3
-  GPL[ -]?v?3\+: GPLv3+
+  GPL[ -]?v?3(?!\.0)?: GPLv3
   LGPL: LGPL
+  LGPL[ -]?v?2(?:\.1)?: LGPLv2.1
   LGPL[ -]?v?3(?:\.0)?: LGPLv3
+exceptions:
+  # Exceptions to avoid correcting SPDX short identifiers
```

I had no idea exception was something possible. It's great.

Pleasure add a link to SPDX here, because many people doesn't know what it could be. And people are too lazy to open the README file.

Please note, you can add an absolute link to the section you added in the README file.

---

I especially like the fact that exceptions avoid the exact string, which means even if the rule has ``ignorecase: true``, it still avoids only the precise string.

I'm unsure about adding an SPDX link since:
1. Most people who see this rule will likely be documentation people who have a higher chance of knowing it.
2. Links are somewhat inconvenient in YAML files.

Still, I recognize your point that most people probably still don't know it. Would this suffice?

```suggestion
  # Exceptions to avoid correcting SPDX identifiers from spdx.org/licenses
```

Also, Do you think adding the licenses page or the main page would be better. The former gets to the point but the latter may be better since the link is added for people who don't have any familiarity with it.

---

I'm fine with your suggestion. 👍

Even, if I'm less optimistic about the fact people will read the documentation 🤭🤣😂